media-sound/pulseaudio-daemon/pulseaudio-daemon-16.0.ebuild

```diff
@@ -78,14 +84,15 @@ COMMON_DEPEND="
 	equalizer? (
 		sci-libs/fftw:3.0
```

Comment:
I think we probably need := on the FFTW deps (it exposes a subslot for the library it installs, it's the library we're using, and we link against it, so we should)